src/browser/fetch/response.rs

```diff
@@ -96,3 +96,48 @@ impl Response {
         &self.raw_response
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use wasm_bindgen_test::*;
+    wasm_bindgen_test_configure!(run_in_browser);
+    use super::*;
+    use serde::Deserialize;
+
+    #[derive(Deserialize)]
+    struct Obj {
+        key: String,
+    }
+
+    #[wasm_bindgen_test]
+    async fn test_json() {
```

Comment:
Change test names please - e.g. `test_json` to `response_json`.
It will be more expressive while you are looking at the testing log and especially it allows you to run all response tests by prefix - e.g. `cargo make test_one_h response_`